kyo-bench/src/main/scala/kyo/bench/Bench.scala

```diff
@@ -42,6 +42,8 @@ end Bench
 
 object Bench:
 
+    inline given kyo.kernel.Frame = kyo.kernel.Frame.internal
```

Comment:
There are some situations where an internal frame is necessary. `Frame.internal` is an `private[kyo]` API that allows generating a frame within the `kyo` package.